### ISSUE TITLE
What is the Whitelist Format?

### ISSUE THREAD

**Issue by btg611** [Vurv78/WebAudio#57]
Hi,

I'm trying to add broadcastify to the whitelist on my server, but I can't seem to figure out the format for the webaudio_whitelist.txt.
Is it the same is what's in the LUA files?

What exactly does the pattern format do?
		pattern %w+%.sndcdn%.com
As opposed to the simple?
		simple translate.google.com

I added this to the webaudio.lua file to no avail:
	-- Broadcastify
	simple [[https://www.broadcastify.com]],
and 
	simple https://www.broadcastify.com
to webaudio_whitelist.txt, also no success.

The links I desire to use look like this:
https://www.broadcastify.com/scripts/playlists/1/20906/-6461706680.m3u

Any help would be appreciated.

**Comment by Vurv78** [comment]
> Is it the same is what's in the LUA files?  

Yes, it's the same as in the lua file, except missing the quotes / ``[[ ]]`` square brackets.

> What exactly does the pattern format do?

The pattern version uses lua patterns (so you can match urls that vary slightly). you shouldn't have to worry about that unless the url you are trying to whitelist has multiple possible urls like some apis do.

And your issue here is that you're including the https:// and the www. which is unnecessary, you can see from the ``simple translate.google.com`` you just need the part after the www.

So try
``simple broadcastify.com`` in your custom file or ``simple [[broadcastify.com]]`` in the lua file

Then again I could probably just add this to the default whitelist myself it does seem to be a pretty large website

**Comment by btg611** [comment]
I see, thanks for the information.  This is very helpful!

I would really appreciate adding the website to the default whitelist, if it's not too much trouble.
